index.php

```diff
@@ -1,5 +1,35 @@
 <?php
-require_once($_SERVER['DOCUMENT_ROOT'] . '/includes/theme-compat/header.php');
+require_once (dirname(__FILE__).'/includes/required-includes.php'); //Necessary file containing: The necessary custom functions, Config file constants, DB connection
+
+//params for include template
+$meta_title = "Home Page | JobriX.tk";
+$page_name = 'home';
+
+//header template include
+require_once (dirname(__FILE__).'/includes/theme-compat/header.php');
+
+
+
+$sql = "SELECT COUNT(status)
+FROM jobs
+WHERE status = 'active'; ";
+
+$result = db_sql_run($sql);
+$result = mysqli_fetch_assoc($result);
+$active_jobs = $result['COUNT(status)'];
+
+$start_index = 0;
+
+
+function jobs_start_index_for_page($page)
```

Comment:
we have mixed coding styles, are we going to have
`function my_func() {`
or 
```
function my_func() 
{
```
we have to use the same styling

Reply:
Everywhere in my code I use this style:
function my_func() 
{

Reply:
I can find ~20 instances of the other way around in this PR only

I would strongly suggest to start using `function my_func() {` as WordPress would require the following coding styling

Reply:
I added in IDE, WordPress coding standards and reformatted all the code of the project.